upload_movie.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import os
+import pandas as pd
+import re
+import logging
+from imdb import IMDb
+from multiprocessing import Pool
+from pymongo import MongoClient
+
+
+RELEVANT_KEYS = ['imdbID', 'title', 'genres', 'votes', 'plot', 'synopsis']
```

Comment:
This also shoud go in a constants.py file
The reason is that it can be reused in scripts etc, it is not necessarily relevant for this file only